extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -174,15 +181,66 @@ public QuartzScheduler(SchedulerContext context, QuartzSupport quartzSupport, Sc
                                         break;
                                 }
                             }
-                            scheduleBuilder = CronScheduleBuilder.cronSchedule(cron);
+                            CronScheduleBuilder cronScheduleBuilder = CronScheduleBuilder.cronSchedule(cron);
+                            if (runtimeConfig.namedJobs.containsKey(identity)) {
```

Comment:
Maybe we could just do a `.get(identity)` and check for null. 
This way we'd have the value of the policy already which we could reuse throughout.
What do you think?